app/scripts/administration/directives/location-authorization.directive.js

```diff
@@ -142,6 +156,19 @@
     }
 
 
+    function selectAllAggregateLocations() {
+      if ($scope.selectedLocationsAggregateData.selectedAllAgg === false) {
+        $scope.selectedLocationsAggregateData.selectedAllAgg = true;
+        $scope.selectedLocationsAggregateData.locations = $scope.locations;
+      }
+      else {
+        $scope.selectedLocationsAggregateData.selectedAllAgg = false;
```

Comment:
is this function wrongly named? Otherwise why calling this function may result in selectAllAgg being set to false?